src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
```

Comment:
Let's move this to a lifecycle method. There is no guarantee of execution order here. It's possible the appropriate filter hasn't yet registered.